src/tools/cardos-tool.c

```diff
@@ -1182,6 +1182,9 @@ int main(int argc, char *argv[])
 			util_print_usage_and_die(app_name, options, option_help, NULL);
 		}
 	}
+	
+	if(action_count == 0)
```

Comment:
please, try to keep the formatting as in the rest of the code:
```suggestion
	if (action_count == 0)
```